### PR TITLE
Add xStation5 v2.34.0-Build.3

### PR DESCRIPTION
**Pull request by mtsz-pl**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/xstation5.rb [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+cask "xstation5" do
+  version "2.34.0-Build.3"
+  sha256 :no_check
+
+  url "https://www.xtb.com/pl/xstation5/xStation5-MacOS.dmg"
+  name "xStation5"
+  desc "Desktop trading platform"
+  homepage "https://www.xtb.com/"
+
+  app "xStation5.app"
```
Comment (neutric) [inline Casks/xstation5.rb:13]:
```
livecheck do
  url "https://xs5.xopenhub.pro/xtb_prod/desktop/mac/latest-mac.yml"
  strategy :electron_builder
end
```

**Review by neutric — COMMENTED** [review]
Thank you for your contribution to `homebrew-cask` @mateuszjablonski!

Please try to add a [`zap` stanza](https://docs.brew.sh/Cask-Cookbook#stanza-zap). The linked documentation provides some recommended methods to create it.

**Comment by mateuszjablonski** [comment]
@neutric I have followed the instruction on https://docs.brew.sh/Cask-Cookbook#stanza-zap and it looks like there is nothing to zap here.

**Comment by neutric** [comment]
@mateuszjablonski Did you start the app so that it can create its files? I get

```
zap trash: [
  "~/Library/Application Support/xStation5",
  "~/Library/Logs/xStation5",
  "~/Library/Preferences/xStation5.desktop.x64.plist",
  "~/Library/Saved Application State/xStation5.desktop.x64.savedState",
]
```

**Comment by mateuszjablonski** [comment]
@neutric I have opened the app, then rechecked the locations and indeed those files are there. I have updated the PR. Thanks for help.

**Comment by bevanjkay** [comment]
@mateuszjablonski Thank you for your contribution to `homebrew-cask` 🎉

**Pull request merged**